### PR TITLE
 Fixed emptyCell not working after prosemirror-model@1.7.1 update

### PR DESCRIPTION
**Pull request by fredguile**
The following commit on `prosemirror-model@1.7.1` has made explicit that we're doing something wrong in our `emptyCell()`, as we can't pass a `Slice` to `tr.replaceWith()`, it must be a `Node` or `Fragment`.

https://github.com/ProseMirror/prosemirror-model/commit/8bf01f1b58e81ffabf64e2944538fca80fc578d8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,14 +53,14 @@
     "prosemirror-tables": "^0.7.9"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.5.4",
```
Comment (RatoX) [inline package.json]: To do it we need to update the babel version, this is much more work and I think it's not part of that PR, maybe we should step back this and focus only on the `emptyCell` fix, what do you think?

File: src/table.js [hunk 2]
```diff
@@ -324,13 +323,9 @@ export const selectTable = tr => {
...
   if (cell) {
-    const content = tableNodeTypes(schema).cell.createAndFill().content;
+    const { content } = tableNodeTypes(schema).cell.createAndFill();
     if (!cell.node.content.eq(content)) {
-      tr.replaceWith(
-        cell.pos,
-        cell.pos + cell.node.nodeSize - 1,
-        new Slice(content, 0, 0)
-      );
+      tr.replaceWith(cell.pos + 1, cell.pos + cell.node.nodeSize, content);
```
Comment (RatoX) [inline src/table.js]: Why did you remove the `-1` on the second parameter? because of the `nodeSize` works like length and the `replaceWith` needs a index.

File: .babelrc [hunk 3]
```diff
@@ -1,14 +1,10 @@
 {
   "env": {
     "testing": {
-      "presets":[
-        ["env"]
-      ]
+      "presets": [["@babel/preset-env"]]
```
Comment (RatoX) [inline .babelrc]: not sure if we can do that now.

File: src/table.js (hunk 2 above)
Comment (RatoX) [reply src/table.js]: and I don't think the `cell.pos+1` will work too, could you write tests for it?

Comment (fredguile) [reply src/table.js]: That's the beauty: tests are already in place, and they fail unless you offset these positions by 1. I think we formerly targeted the tableCell, now it targets the tableCell's content.

File: package.json (hunk 1 above)
Comment (fredguile) [reply package.json]: I got issues having jest transpiling the sources, but you're right and @d4rkr00t also suggested to put this in another PR... so I've removed this commit from this PR.

**Comment by RatoX** [comment]
@fredguile this will be a major change since we will need the prosemirror-model version 1.7.1 to this work.

Could you update the verison on the package.json too?

`"prosemirror-model": "^1.0.0", `

**Comment by fredguile** [comment]
> @fredguile this will be a major change since we will need the prosemirror-model version 1.7.1 to this work.
> 
> Could you update the verison on the package.json too?
> 
> `"prosemirror-model": "^1.0.0", `

Let's reason about it. `prosemirror-model@1.7.1` doesn't introduce any behavioural change, except that it now throws if we pass a wrong argument to `tr.replaceWith`. The PR also doesn't introduce any behavioural change, just fixes the wrong argument, so that should be backward compatible with  `prosemirror-model@^1.0.0"` right?
